platform/src/test/resources/action/path_payment_operation_record.json

```diff
@@ -0,0 +1,100 @@
+[
```

Comment:
There are many json files for tests. It would be easier to maintain if these json strings are defined in the same file as the tests unless they are used in multiple tests.

Reply:
Fixed in https://github.com/stellar/java-stellar-anchor-sdk/pull/1082.
I left 7 files, that are used in 2+ test classes:
![image](https://github.com/stellar/java-stellar-anchor-sdk/assets/131388170/cd4f1d52-d9fd-4115-a80a-c9bc2b66cbb0)